fec_eregs/docs/Team Practices.md

```diff
@@ -0,0 +1,143 @@
+Trello will be our used as our taskboard.  All work the team is doing should be reflected on the taskboard.  Backlog grooming/planning/estimation/demos will be driven by the task board.  
+
+Owners of each card should comment frequently to provide visibility to progress (or lack thereof).  
+
+While the product steward will likely be creating the bulk of the cards (in collaboration with the FEC product owner), any team member can create cards that will be prioritized into grooming sessions.
+
+TBD - Creating / Syncing cards from whatever forum will be used for user feedback, reported defects etc.
+
+## GitHub
+Our code repositories.  
+The following are GH repositories of note:
+* [fec-eregs](https://github.com/18F/fec-eregs)
+* [regulations-core](https://github.com/18F/regulations-core)
+* [regulations-site](https://github.com/18F/regulations-site)
+* [regulations-parser](https://github.com/18F/regulations-parser)
+* [openFEC-web-app](https://github.com/18F/openFEC-web-app)
+* [fec-styles](https://github.com/18F/fec-styles)
+* [fec-cms](https://github.com/18F/fec-cms)
+
+TBD - GH Issues syncing wih Trello
+
+## Slack
+Given our distributed model, slack will be used for async comms.  The following are channels of note:
+* #fec-eregs
+* #fec-general
+* #fec-partners
+* #eregs
+
+# Sprint Cadence
+Sprints will be used to time box our development cycles.  
+Sprints will be two weeks long.  
+It will start with Backog grooming and planning, and end with Demos and Retrospectives.
+
+## Backlog Grooming/Planning
+At the beginning of a sprint a prioritized list of activities will be present on the team taskboard.
+This meeting will be used to review those stories where we will:
+* Gain a common understanding of what the stories entail
+* Ensure the team is in a position to execute the story (any dependencies)
+* Commit to which stories the team will complete for that given Sprint
+
+### Estimation
+Storypointing in Agile is used in conjunction with velocity to project how much a team can commit to.  However, given our team is nascent in nature, establishing the velocity is a second priority.
+Estimation is going to be used to ensure that as we discuss stories, there is a common understanding as to what it entails.  
+One of the best sources to find disconnects and align on them is by each member of the team pointing.
+Further, pointing gives a common baseline which can drive whether a story should be broken down further into smaller chunks.
+
+There are at least 2 trains of thought in terms of units of measure for storypoint: complexity vs. effort (time).  
+The difference between the two is described through the example of tasks that are great candidates for automation.  Those tasks often carry high effort estimates, but very low complexity estimates.
+Given the forming nature of the team, effort estimates will be used.  
+Aligning on a baseline of complexity may come at a later time.
+
+*Units of Measure*
```

Comment:
Agreed, whether effort or complexity based, the points should be used in relativity rather than absolute units.  Given we went through a pass of grooming yesterday, we now have something we can use as a base reference.  Will remove.